README.md

```diff
@@ -1,87 +1,69 @@
+(por enquanto). Como executamos tudo dentro de contêineres, você só precisa instalar o [`podman`]
+ (https://podman.io/) para rodar localmente em sua máquina. Se você não está familiarizado com o 
+ `podman`, você pode pensá-lo como um `docker` /` docker-compose` mais “leve”. Ele existe em 
+ quase todos os repositórios de pacotes de distribuição Linux.     
+Por favor, consulte a  [documentação do podman](https://podman.io/getting-started/installation
+.html) para entender como instalá-lo em seus ambientes. Se tiver alguma dificuldade com o podman, fale com a gente! 
 
-In order to be able to run and test your changes in the project, first, you need
-to build the container image used during development. For that you can use the
-following command:
+## Instalação
+
+Para poder executar e testar suas alterações no projeto, primeiro você precisa construir a imagem
+ do contêiner usado no desenvolvimento. Para isso, use o seguinte comando: 
 
 ```
 make build
 ```
 
-After that you will have a container image which can be use to run the API 
-locally during development.
+Com isso, você terá uma imagem do contêiner que poderá executar a API localmente no processo de desenvolvimento. 
 
-## Running
+## Executando
 
-To run the API locally in your machine, you can run the following command:
+Para executar a API localmente em sua máquina, use o seguinte comando:
 
 ```
 make run
 ```
 
-This command will start all containers necessary to run the API. In other words,
-it starts the database and the API container. If everything goes fine, you 
-should be able to query the API at `localhost:8080/gazettes/<City IBGE Code>`
-
-NOTE: When you want to restart the API, just quit the API process and 
-execute `make rerun` again. You do not need to restart the database.
+Esse comando vai iniciar todos os contêineres necessários para executar a API. Ou seja, ele inicializa o banco de dados e o contêiner da API. Se tudo correr bem, você poderá fazer consultas à API em `localhost:8080/gazettes/<City IBGE Code>`
 
-You can all check the interactive documentation at `localhost:8080/docs`. Using 
-the docs page, you can all send request to the API. But to see it working you
-need to insert data into the database. There is another make target, `make apisql`,
-which open the `psql` and connect to the database. Thus, you can insert data
-using some `INSERT INTO ...` statements and test the API. ;)
+ATENÇÃO: Quando você precisar reiniciar a API, apenas interrompa o processo da API e execute o `make rerun` novamente. Não é necessário reiniciar o banco de dados.
+Você pode checar toda a documentação interativa da API em  `localhost:8080/docs`. Nessa página, você pode fazer requisições à API diretamente. Mas, para vê-la funcionar, você precisa inserir dados no banco de dados. Há outro comando, `make apisql`, que abre o `psql` e conecta ao banco de dados. Assim, você pode inserir dados usando alguns `INSERT INTO ...` para testar a API  ;)
```

Comment:
Aqui repito o comentário que fiz antes sobre o novo parágrafo.

---
### English

Here I repeat the same comment I made about the new paragraph.